README.md

```diff
@@ -1,110 +1,97 @@
 
-### Dependencies
+### Large Populations (Brute force)
 
-The dependencies below are required for building FLAME GPU 2.
+This figure shows how the average simulation time scales with ensemble size for different large population sizes of a brute force messaging Boids model.
+	+ Population sizes are stepped between 2048 and 8192 at intervals of 2048
+	+ Simulation timing is measured over 500 steps
 
-+ [CMake](https://cmake.org/download/) `>= 3.18`
-+ [CUDA](https://developer.nvidia.com/cuda-downloads) `>= 11.0` and a Compute Capability `>= 3.5` NVIDIA GPU.
-  + CUDA `>= 10.0` currently works, but support will be dropped in a future release.
-+ C++17 capable C++ compiler (host), compatible with the installed CUDA version
-  + [Microsoft Visual Studio 2019](https://visualstudio.microsoft.com/) (Windows)
-  + [make](https://www.gnu.org/software/make/) and [GCC](https://gcc.gnu.org/) `>= 7`(Linux)
-  + Older C++ compilers which support C++14 may currently work, but support will be dropped in a future release.
-+ [git](https://git-scm.com/)
+![sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop_brute_force.png](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop_brute_force.png)
 
-### Building FLAME GPU 2
+### Large Populations (Spatial)
 
-FLAME GPU 2 uses [CMake](https://cmake.org/), as a cross-platform process, for configuring and generating build directives, e.g. `Makefile` or `.vcxproj`. This is used to build the FLAMEGPU2 library, examples, tests and documentation.
+This figure shows how the average simulation time scales with ensemble size for different large population sizes of a spatial messaging Boids model.
+	+ Population sizes are stepped between 2048 and 8192 at intervals of 2048
+	+ Simulation timing is measured over 500 steps
 
-Below the core commands are provided, for the full guide refer to the main [FLAMEGPU2 guide](https://github.com/FLAMEGPU/FLAMEGPU2/blob/master/README.md).
+![sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop.png](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop.png)
 
-#### Linux
+## Building and Running the Benchmark
 
-Under Linux, `cmake` can be used to generate makefiles specific to your system:
+Detail of dependencies and the `cmake` build process are described in full in the [FLAMEGPU2-example-template Repo](https://github.com/FLAMEGPU/FLAMEGPU2-example-template) and are not repeated here. The benchmark should be built with seatbelts off (e.g. `-DSEATBELTS=OFF` passed to the `cmake` configuration step) to disable additional run-time checks and optionally disabling Python Swig support which is not needed for this model(-DBUILD_SWIG_PYTHON=OFF ). E.g. for Volta (`SM_70`) GPUs under Linux.
 
 ```bash
-mkdir -p build && cd build
-cmake .. 
-make -j8
+# Configure 
+cmake . -B build -DCMAKE_BUILD_TYPE=Release -DSEATBELTS=OFF -DBUILD_SWIG_PYTHON=OFF -DCUDA_ARCH=70
+cmake --build build -j`nproc` 
+make
```

Comment:
`make` is incorrect / should be removed. `cmake --build` does this + no makefile in the repository root.